package.json

```diff
@@ -7,20 +7,31 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "upgrade-react-page": "for i in react-page editor plugins-background plugins-slate; do yarn add \"@react-page/${i}@latest\"; done"
+    "upgrade-react-page": "export PACKAGE=''; for i in react-page editor plugins-background plugins-slate plugins-image plugins-spacer; do PACKAGE=\"${PACKAGE} @react-page/${i}@latest\"; done;  yarn add ${PACKAGE}"
   },
   "dependencies": {
-    "@react-page/editor": "^0.26.0",
-    "@react-page/plugins-background": "^0.26.0",
-    "@react-page/plugins-slate": "^0.26.0",
-    "@react-page/react-page": "^0.26.0",
+    "@react-page/editor": "^0.27.0",
+    "@react-page/plugins-background": "^0.27.0",
+    "@react-page/plugins-image": "^0.27.0",
+    "@react-page/plugins-slate": "^0.27.0",
+    "@react-page/plugins-spacer": "^0.27.0",
+    "@react-page/react-page": "^0.27.0",
```

Comment:
we can remove the meta package @react-page/react-page"

Reply:
done